tests/unit/DetailsList.spec.ts

```diff
@@ -19,7 +19,7 @@ describe('Details List', () => {
     title: 'Annual Report (2018)',
     filingId: 63958,
     submitter: 'John Doe',
-    submittedDate: '2018-12-12',
+    submittedDate: 'Wed, 12 Dec 2018 12:00:00 GMT',
```

Comment:
The old data is not a valid GMT string. The new data is.

Reply:
Explanation:

Submitted Date (and Effective Date), both in the filing data from the Legal API, are GMT strings (as above).

Corrected Filing Date is based on the submitted date of the filing being correct, but Corrected Filing Date must be a JSON "date", such as YYYY-MM-DD.

Correction schema:
https://github.com/bcgov/business-schemas/blob/1d00db747741857c3919e31f9ee37280a16295cb/src/registry_schemas/schemas/correction.json#L57

Defintion of JSON schema date:
![Clipboard - June 15, 2023 10_05 AM](https://github.com/bcgov/business-filings-ui/assets/10002889/f5740495-8325-4027-a2e5-ac0d04d2895b)